src/hooks/useConnectedProvidersLite.js

```diff
@@ -56,7 +57,12 @@ const useConnectedProvidersLite = (internalId, type, shouldExecute = true) => {
    * @returns {Void} None.
    */
   const filterProviders = (response) => {
-    let providerAssets = response.filter((item) => item.connected && type.includes(item.type));
+    let providerAssets = null;
+    if (onlyConnected) {
+      response.filter((item) => item.connected && type.includes(item.type));
+    } else {
+      response.filter((item) => type.includes(item.type));
```

Comment:
We should avoid repeating code (DRY), maybe like that:
```
response.filter((item) => (!onlyConnected || item.connected) && type.includes(item.type));
```
But anyway `filter()` doesn't mutate the data so it's not doing anything?

Reply:
Done.